archeo/inc/patterns/image-with-headline-description.php

```diff
@@ -11,8 +11,8 @@
 	<h2 class="has-text-align-right" id="chahk-raindeity" style="font-size:clamp(64px, 6vw, 100px);line-height:1;margin-bottom:48px;text-transform:uppercase">' . wp_kses_post( __( 'Chahk:<br>Rain<br>deity', 'archeo' ) ) . '</h2>
 	<!-- /wp:heading -->
 
-	<!-- wp:columns -->
-	<div class="wp-block-columns"><!-- wp:column {"width":"15%"} -->
+	<!-- wp:columns {"style":{"spacing":{"margin":{"bottom":"0px"}}}} -->
```

Comment:
I am not sure this quite gets it there. I think the spacing is now the paragraph's default bottom margin, which is 1em: 

<img width="1903" alt="Screen Shot 2022-03-09 at 2 40 35 PM" src="https://user-images.githubusercontent.com/5375500/157520366-4c15d7d6-9942-462d-910c-428f6406ffb7.png">

We'd either have to overwrite the paragraph's bottom margin there, or add some arbitrary margin value to the columns to get the sum to be `48px`.

Reply:
I pushed a couple small changes that I think get this closer. I adjusted the value of the `medium` custom spacing value, but it didn't appear to have any negative effects elsewhere — I like the extra space it adds. 

I also messed with the bottom margin on the homepage main title there... it's not 100% how I'd like it, but it's pretty close.

Reply:
> I think the spacing is now the paragraph's default bottom margin, which is 1em

I was adding up the spacing above the 'Selected Writings' heading and the bottom of this paragraph to `48px`. I must've misunderstood!

> I adjusted the value of the medium custom spacing value

I think this looks good and doesn't seem to negatively affect anything else. Thanks!